cves/CVE-2011-2729.yml

```diff
@@ -162,11 +162,11 @@ lessons:
     applies:
     note:
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: Allows improper access privlidges
```

Comment:
spelling mistake, privileges

Reply:
Thanks for catching that spelling mistake I will update it right away.